apollo-router/src/configuration/migrations/0037-preview_otlp_error_metrics.yaml

```diff
@@ -0,0 +1,6 @@
+description: The OTLP error metrics option is now in the preview release stage
+actions:
+
+  - type: move
+    from: telemetry.apollo.experimental_otlp_error_metrics
```

Comment:
There's a typo here as well - it should probably be 
```
    from: telemetry.apollo.errors.experimental_otlp_error_metrics
    to: telemetry.apollo.errors.preview_extended_error_metrics
```

but even with that I'm getting a config error on startup when i try to use this config:

```
telemetry:
  apollo:
    errors:
      experimental_otlp_error_metrics: enabled
```

Can you see if you can get it working?